grpc/cpp-cubic/README.md

```diff
@@ -38,7 +38,25 @@ copy this repository into your project and add the line
 add_subdirectory(sdk-cubic/grpc/cpp-cubic)
 ```
 
-to your project's CMakeLists.txt.
+to your project's CMakeLists.txt. You may also include it
+using the [FetchContent module](https://cmake.org/cmake/help/latest/module/FetchContent.html),
```

Comment:
It feels like this could be made into two separate sub-sections.  Like `### Include via local copy` and `### Include via FetchContent`.  Thoughts?  Is that silly?

Reply:
I wouldn't call it silly, but I don't particularly feel like it is necessary.